wiki/Community/Projects/fr.md

```diff
@@ -1,5 +1,6 @@
 ---
-outdated: true
+outdated_since: f31678789896bc4d36e44abb8d89925fb095dfb8
+outdated_translation: true
```

Comment:
you are changing the meaning from "we DON'T know since when the translation is outdated" to "we DO know since when the translation is outdated", ignoring many other potential changes that need to be backported

I get that you added this in attempt to comply with the CI check step, but it is still wrong